AdvancedStateAndSideEffectsCodelab/app/src/main/java/androidx/compose/samples/crane/details/DetailsActivity.kt

```diff
@@ -89,16 +88,14 @@ class DetailsActivity : ComponentActivity() {
         WindowCompat.setDecorFitsSystemWindows(window, false)
 
         setContent {
-            ProvideWindowInsets {
-                CraneTheme {
-                    Surface {
-                        DetailsScreen(
-                            onErrorLoading = { finish() },
-                            modifier = Modifier
-                                .statusBarsPadding()
-                                .navigationBarsPadding()
-                        )
-                    }
+            CraneTheme {
+                Surface {
+                    DetailsScreen(
+                        onErrorLoading = { finish() },
+                        modifier = Modifier
+                            .statusBarsPadding()
+                            .navigationBarsPadding()
```

Comment:
Is just `systemBarsPadding()` enough here?

Reply:
Oh yes actually systemBarsPadding includes status+nav+caption, TY :)